extensions/amp-video-docking/0.1/amp-video-docking.js

```diff
@@ -1050,6 +1050,7 @@ export class VideoDocking {
       // Since the AMP element container is position: relative, the media
       // element is relative to AMP element corner when position: absolute, so
       // we need to offset it by outer container's position.
+      // TODO: DO NOT SUBMIT. This is buggy
```

Comment:
whats the bad bug?

Reply:
`PositionObserver` was somehow passing the rect of the docked e.g. `<video>` element instead of the inline wrapper e.g. `<amp-video>`, which threw off `left/top` in a way that the docked video disappeared briefly.

BUT since we can now use it, I dropped in a `IntersectionObserver` instead and it just works.

Reply:
I was way off here: I was using dimensions for a different registered element: fixed on 353c14a